src/index.tsx

```diff
@@ -20,6 +20,14 @@ if (window.location.origin.includes("explorer.devnet.aptos.dev")) {
   window.location.replace(new_location);
 }
 
+if (window.location.origin.includes("explorer.aptoslabs.com/proposals")) {
```

Comment:
do we need it? we will deprecate the governance pages, so there will be no way to enter those pages on the client side, and we already handle the redirect on the server side

Reply:
oh right I forgot that we've handled it on the serverside